style-desk.css

```diff
@@ -680,4 +680,121 @@
       color: white;
     }
   }
+
+  #popUp {
```

Comment:
- [ ] You design should be similar to Figma design in terms of sizing of elements in the popup. Let's make similar to Figma design. 
- [ ] Your popup design should not have scroll in the description. Let's fix this issue. Cheers 👍 


Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/179870666-3e2fd2c0-d999-4210-bf2f-605574fb6c7c.png)  |  ![image](https://user-images.githubusercontent.com/60618877/179870703-bb3fa9e9-57eb-4358-ba43-8656c0b6fb6a.png)